app/components/footer-boxes/component.js

```diff
@@ -16,6 +17,13 @@ export default Ember.Component.extend({
 
     click() {
         let type = this.get('selected') ? null : this.get('type');
+
+        const category = 'homepage';
+        const action = 'click';
+        const label = type;
+
+        Ember.get(this, 'metrics').trackEvent({ category, action, label });
```

Comment:
This can just be `this.get('metrics')` AFAIK they work the same way. `Ember.get` is mostly intended for things that are just normal objects, not Ember objects.